core/simpleconsumer.go

```diff
@@ -31,6 +32,8 @@ import (
 //    Enable: true
 //    ID: ""
 //    ShutdownTimeoutMs: 1000
+//    ModulatorQueues: 0
```

Comment:
Rename this setting to `ModulatorRoutines` - That should be easier to understand.
How discussed: Use one queue instead of n to avoid starvation.